### DISCUSSION TITLE
Hitting the max rate limit of 1 Message

### DISCUSSION THREAD

**Discussion by sni89** [theyosh/TerrariumPI#727] [Q&A] [answered]
Hey guys, 

just wondering how to enable more than one message per minute on my Oled. I have 1 message (system summary) enabled. But I am still getting  the warning: 

“Hitting the max rate limit of 1 messages per minute for message System” 

message per Minute is set to 1

Edit: I found it. Just set 0 for infinite messages. Bless the ones that are able to read 😇

**Comment by theyosh** [comment, marked as answer]
Hi,

the display notifications are not really working well. Due to small screens and long messages, it can happen that it cannot keep up with the notifications.

So I added it, as I had some hardware laying around. But I think it is not very practical. Most people I advise to make a small script which gather all the data needed through the TerrariumPI API and write your own logic on how to display.
